lib/trivial.nix

```diff
@@ -31,6 +68,18 @@ rec {
   /* boolean “and” */
   and = x: y: x && y;
 
+  /* bitwise “and” */
+  bitAnd = builtins.bitAnd or zipIntBits (a: b: if a==1 && b==1 then 1 else 0);
+
+  /* bitwise “or” */
+  bitOr = builtins.bitOr or zipIntBits (a: b: if a==1 || b==1 then 1 else 0);
+
+  /* bitwise “xor” */
+  bitXor = builtins.bitXor or zipIntBits (a: b: if a!=b then 1 else 0);
+
+  /* bitwise “not” */
+  bitNot = builtins.sub (-1);
```

Comment:
When I try this in  `nix repl`, it  doesn't find `bitNot`. Probably because it's not listed in `default.nix`.